### PR TITLE
Docs  add recipe to use store with elements

### PR DESCRIPTION
**Pull request by santoshyadav198613**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: projects/ngrx.io/content/guide/store/recipes/angular-elements.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Using Store with Angular Elements
+
+## Creating Angular Elements
+
+The following recipe illustrates utilizing Angular Elements with a NgRx Store. This provides the ability to manage the state of a counter, and to select and display that counter state from within separate Angular Elements. This is all accomplished from within a single Angular Project and static HTML page.
```
Comment (jtcrowson) [inline projects/ngrx.io/content/guide/store/recipes/angular-elements.md]:
```suggestion
The following recipe illustrates utilizing [Angular Elements](https://angular.io/guide/elements) with a NgRx Store. This provides the ability to manage the state of a counter, and to select and display that counter state from within separate Angular Elements. This is all accomplished from within a single Angular Project and static HTML page.
```

**Comment by timdeschryver** [comment]
It would be great to re-use the counter app, but on the other hand it might over-complicate the counter example.

File: projects/ngrx.io/content/guide/store/recipes/angular-elements.md (hunk 1 above)
Comment (santoshyadav198613) [reply projects/ngrx.io/content/guide/store/recipes/angular-elements.md]: In the actual example, we wanted to have a sepearet element for every action. Agreed with empty css files let me remove.

**Comment by santoshyadav198613** [comment]
Removed Css and updated doc as per comment

**Comment by santoshyadav198613** [comment]
Hi @timdeschryver and @jtcrowson let me know if we need to change something.

**Comment by jtcrowson** [comment]
`projects/ngrx.io/content/examples/store-elements/projects/my-counter/README.md` can be deleted as well.

I'll try running through the recipe completely tomorrow.

**Comment by santoshyadav198613** [comment]
> `projects/ngrx.io/content/examples/store-elements/projects/my-counter/README.md` can be deleted as well.
> 
> I'll try running through the recipe completely tomorrow.

Done. please check.

**Comment by santoshyadav198613** [comment]
Hi @jtcrowson ,
can you check once.

**Comment by santoshyadav198613** [comment]
Hi @jtcrowson and @timdeschryver ,
can you check once.

**Comment by brandonroberts** [comment]
Just reopen this PR when you're ready to pick it back up. Thanks!

**Pull request closed without merging**